store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/FederatedStorePropertiesTest.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2024 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.federatedstore;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import uk.gov.gchq.gaffer.commonutil.StreamUtil;
+
+import java.io.InputStream;
+import java.net.URISyntaxException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.gchq.gaffer.federatedstore.FederatedStoreTestUtil.CACHE_SERVICE_CLASS_STRING;
+
+public class FederatedStorePropertiesTest {
+    final String key = "gaffer.store.class";
+    final String value = FederatedStore.class.getName();
+    static Path propertiesFilePath;
+
+    @BeforeAll
+    public static void init() throws URISyntaxException {
+        propertiesFilePath = Paths.get(FederatedStorePropertiesTest.class.getResource("/properties/federatedStore.properties").toURI());
+    }
+
+    @Test
+    public void shouldGetIsPublicAccessAllowed() {
```

Comment:
Just a tip, don't need `public` modifiers on test methods or classes in Junit

Reply:
I tend to keep consistent with this for existing classes. But as this is a new class, I'll fix it to not have it. I think it's there as I copy pasted the existing StoreProperties unit test to use as a template...